src/components/Support/PartnerSection/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { SupportSection } from "../SupportSection";
+import { PartnerCard } from "./PartnerCard";
+
+export const PartnerSection = () => (
+  <SupportSection
+    id="partner-section"
+    title="Zostań naszym partnerem"
+    card={<PartnerCard />}
```

Comment:
Maybe we could inline this `<PartnerCard />` contents here? as it is only 3 lines long?
No strong feelings here, so I'd like to hear other opinions about that 😅